### ISSUE TITLE
goverlay will not open even after restarting system

### ISSUE THREAD

**Issue by kalebb2001** [benjamimgois/goverlay#19]
I installed mangohud and goverlay and it shows up but everytime I try and open the program it does nothing.

**Comment by Deickar** [comment]
i have de same issue Goverlay dont start i tray using with heroic launcher that error shows me in log : env: error while loading shared libraries: libfmt.so.8: cannot open shared object file: No such file or directory.

mangoHud donset work and dosent work with steam to.

**Comment by suboptimalraccoon** [comment]
Yeah i have the same issue, games with mangohud command on steam wont launch, same as goverlay itself

**Comment by benjamimgois** [comment]
What distro ? Does mangohud works on vkcube ?

**Comment by suboptimalraccoon** [comment]
Okay, everything started working after reinstaling all mangohud components, im on garuda linux based on arch, and used octopi to reinstall mangohud

**Comment by benjamimgois** [comment]
I usually see this kind of problem with ubuntu users that are using a old mangohud release from the ppa.

**Comment by slashgob** [comment]
I had the same issue, opening Goverlay just resulted in it crashing instantly. if i opened from terminal I got this error...

Error of failed request:  BadValue (integer parameter out of range for operation)

I deleted the MangoHud.conf file in ~/.conf/MangoHud and Goverlay opened again. Clicking the save config button created a new config. If I closed Goverlay and re-opened it would crash again.

I examined the config file with the help of some guys on the GE Discord and we found the culprit to be gl_vsync=-1 was causing the error, which is setting vsync to Adaptive. If i edited this to be any other value (like =0 or =1) and saved the config, Goverlay would open again.

So, for some reason it did not like gl_vsync being set to Adapitve.

OS: Nobara Linux 36 (Thirty Six) x86_64 
Host: X570 AORUS PRO -CF 
Kernel: 5.19.12-201.fsync.fc36.x86_64  
Packages: 2713 (rpm), 10 (flatpak) 
Shell: bash 5.1.16 
Resolution: 3440x1440 
DE: GNOME 42.3.1 
WM: Mutter 
WM Theme: Adwaita 
Theme: adw-gtk3-dark [GTK2/3] 
Icons: Sweet-Rainbow [GTK2/3] 
Terminal: kitty 
CPU: AMD Ryzen 7 5800X (16) @ 3.800GHz 
GPU: NVIDIA GeForce GTX 1080 Ti 
Memory: 4615MiB / 15914MiB

**Comment by benjamimgois** [comment]
> I had the same issue, opening Goverlay just resulted in it crashing instantly. if i opened from terminal I got this error...
> 
> Error of failed request: BadValue (integer parameter out of range for operation)
> 
> I deleted the MangoHud.conf file in ~/.conf/MangoHud and Goverlay opened again. Clicking the save config button created a new config. If I closed Goverlay and re-opened it would crash again.
> 
> I examined the config file with the help of some guys on the GE Discord and we found the culprit to be gl_vsync=-1 was causing the error, which is setting vsync to Adaptive. If i edited this to be any other value (like =0 or =1) and saved the config, Goverlay would open again.
> 
> So, for some reason it did not like gl_vsync being set to Adapitve.
> 
> OS: Nobara Linux 36 (Thirty Six) x86_64 Host: X570 AORUS PRO -CF Kernel: 5.19.12-201.fsync.fc36.x86_64 Packages: 2713 (rpm), 10 (flatpak) Shell: bash 5.1.16 Resolution: 3440x1440 DE: GNOME 42.3.1 WM: Mutter WM Theme: Adwaita Theme: adw-gtk3-dark [GTK2/3] Icons: Sweet-Rainbow [GTK2/3] Terminal: kitty CPU: AMD Ryzen 7 5800X (16) @ 3.800GHz GPU: NVIDIA GeForce GTX 1080 Ti Memory: 4615MiB / 15914MiB

@slashgob  since i pretty much always use vsync ON i haven't meet this issue. Just made a try on my Manjaro testing machine and it works fine on intel / mesa drivers. When i got home i'll try it on Nvidia drivers.
